### DISCUSSION TITLE
Vue Devtools extension not working with electron

### DISCUSSION THREAD

**Discussion by gladbhex** [quasarframework/quasar#9832] [CLI - Electron mode] [answered]
> We’ve removed the default electron-main.dev.js support as it seems that it’s not needed anymore
This is what the documentation tells. I am unable to get the Vue dev tools extension working with the latest release of Quasar(v2) in electron dev mode. Am I missing something?

**Comment by alitalaghat** [comment, marked as answer]
For Quasar v2 (VueJs v3), we need vue-devtools v6.
yarn add --dev "@vue/devtools@6.0.0-beta.19"
